catalog/kyverno/repositories.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta1
+kind: HelmRepository
+metadata:
+  name: kyverno
+  namespace: kyverno
+spec:
+  interval: 120m
```

Comment:
others are specified as  interval: 1h0m0s , I would make everything equal

Reply:
the interval format are now aligned.